test/widgets/compose_box_test.dart

```diff
@@ -134,6 +148,61 @@ void main() {
     await tester.pump(Duration.zero);
   }
 
+  group('auto focus', () {
+    testWidgets('ChannelNarrow, non-empty fetch', (tester) async {
+      final channel = eg.stream();
+      await prepareComposeBox(tester,
+        narrow: ChannelNarrow(channel.streamId),
+        streams: [channel],
+        messages: [eg.streamMessage(stream: channel)]);
+      check(controller).isA<StreamComposeBoxController>()
+        .topicFocusNode.hasFocus.isFalse();
```

Comment:
Can make this check a bit stronger, verifying focus doesn't go to the content input either:
```suggestion
      check(controller).isA<StreamComposeBoxController>()
        ..topicFocusNode.hasFocus.isFalse()
        ..contentFocusNode.hasFocus.isFalse();
```